src/components/App.jsx

```diff
@@ -9,7 +15,11 @@ export const App = () => {
         color: '#010101',
       }}
     >
-      React homework template
+      <Routes>
+        <Route path="/soyummy_frontend" element={<StartPage />} />
```

Comment:
Imagine what happen if I merge this branch before `StartPage` implementation 🤔 
We don't need any error when testing our app so please comment missing variables ❌

Reply:
That's why I added info about the merge, but ok, I'll comment next time

Reply:
ok ✅